### ISSUE TITLE
Adding an embedded codeblock language shouldn't change the root :syn spell toplevel

### ISSUE THREAD

**Issue by cormacrelf** [vim-pandoc/vim-pandoc-syntax#250]
Pandoc syntax sets `:syn spell toplevel`. Some other language syntaxes use `:syn spell notoplevel` to enforce no spelling unless in a comment (even though that is already the behaviour of `default`). But it appears the pandoc's setting is dominated by whichever languages happen to be loaded through  `g:pandoc#syntax#codeblocks#embeds#langs`.

When I open a pandoc document, spelling errors are not highlighted in plain paragraphs. They are in, e.g. list items.

When I ripgrep my plugins directory for `notoplevel`, it only appears in a few files, namely the syntaxes for haskell-vim and vim-scala. Both of these are in my  `g:pandoc#syntax#codeblocks#embeds#langs`. If I remove them, the `syn spell` is not affected and spell checking works correctly.

Basically, it looks like the fix is a one-liner to re-add `syn spell toplevel` after loading the child syntaxes.
